src/graphql/HolofuelCompletedTransactionsQuery.gql

```diff
@@ -1,5 +1,5 @@
-query HolofuelCompletedTransactionsQuery {
-  holofuelCompletedTransactions {
+query HolofuelCompletedTransactionsQuery ($transactionId: String)  {
+  holofuelCompletedTransactions(filters: $transactionId)  {
```

Comment:
this filter variable is not plugged in in the schema and resolvers so won't do anything.

Reply:
Ah!  This is actually residue from when I did have it plugged in.  (I decided not to use this filter for reasons relating to advanced enums within javascript vs Rust. This is one of the items we need to review/include in our api revision during a code refactoring after launch.)

Nonetheless, looks like I deleted it everywhere, but here.  Good catch! :eye: